lib/rodauth/features/jwt.rb

```diff
@@ -26,6 +26,20 @@ module Rodauth
       :use_jwt?
     )
 
+    # JWT claims and verifiers (see ruby-jwt for more info)
+    auth_value_method :jwt_sub, proc { |ra| ra.session[ra.session_key] }
+    auth_value_method :jwt_verify_sub?, false
+    auth_value_method :jwt_iat, proc { Time.now.to_i }
+    auth_value_method :jwt_verify_iat?, true
+    auth_value_method :jwt_jti, nil
+    auth_value_method :jwt_verify_jti?, false
+    auth_value_method :jwt_leeway, 30
```

Comment:
I think this is an OK default, to account for some small amount of clock skew on a small cluster of nodes serving a JSON API, but perhaps it would be better to neither include nor verify an iat claim by default.